app/src/behaviors/behavior_sticky_key.c

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_sticky_key
+
+#include <device.h>
+#include <drivers/behavior.h>
+#include <logging/log.h>
+#include <zmk/behavior.h>
+
+#include <zmk/matrix.h>
+#include <zmk/endpoints.h>
+#include <zmk/event-manager.h>
+#include <zmk/events/position-state-changed.h>
+#include <zmk/events/keycode-state-changed.h>
+#include <zmk/events/modifiers-state-changed.h>
+#include <zmk/hid.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#if DT_NODE_EXISTS(DT_DRV_INST(0))
+
+#define ZMK_BHV_STICKY_KEY_MAX_HELD 10
```

Comment:
I've noticed the the `ZMK_BHV_` prefix in hold tap also.

Suggest we simplify it to `ZMK_` as there's unlikely to be namespace conflicts?

Reply:
I continued this convention from pete's original hold-tap implementation. I don't have a strong opinion on this naming scheme.

Reply:
Other note: Do we really think 10 is a reasonable default for this?

Reply:
ah, I didn't spot that they're local.  In which case they probably don't even need a prefix.  I'm not fussed.

10 seemed like an absolute maximum to me.  We could probably get away with less without any issues though?

Reply:
As written, this shares the sticky state between sticky keys and layers, which may explain why it is larger. Will wait for @okke-formsma to weigh in.